src/cohort/dto/cohort-search.dto.ts

```diff
@@ -1,8 +1,28 @@
 import { ApiProperty, ApiPropertyOptional } from "@nestjs/swagger";
-import { IsBoolean, IsNumber, IsNumberString, IsObject, IsOptional, IsString } from "class-validator";
+import { IsBoolean, IsNotEmpty, IsNumber, IsNumberString, IsObject, IsOptional, IsString, IsUUID, ValidationArguments, ValidationOptions, registerDecorator } from "class-validator";
 import { CohortDto } from "./cohort.dto";
 import { Expose } from "class-transformer";
 
+// Custom decorator to check if the object is not empty
+function IsNotEmptyObject(validationOptions?: ValidationOptions) {
```

Comment:
@souravbhowmik1999 Check this all code is really needed to check non empty object